src/ssl.c

```diff
@@ -15821,6 +15821,7 @@ int wolfSSL_set_compression(WOLFSSL* ssl)
         ctx->x509_store.cache = str->cache;
         ctx->x509_store_pt    = str; /* take ownership of store and free it
                                         with CTX free */
+        ctx->cm->x509_store_p = ctx->x509_store_pt;
```

Comment:
Where is ownership and control documented?

Reply:
Thank you for pointing this out. Added comments [here](https://github.com/miyazakh/wolfssl/blob/fae36f108eef40fb99bcce894c22b08047b1ef37/wolfssl/internal.h#L2086-L2088)